ChangeLog.md

```diff
@@ -18,8 +18,15 @@ to browse the changes between the tags.
 
 See docs/process.md for more on how version tagging works.
 
+Note: If you are reading this file as part of a git checkout the topmost version
```

Comment:
It seems even more likely that this file will be read directly from github. so maybe this could say "reading this file from the main branch on github or a git checkout" or something like that.

Reply:
I just reverted this part and we can land something like this as part of a fix for #14726